pkgs/applications/graphics/xournal/default.nix

```diff
@@ -2,7 +2,9 @@
 , ghostscript, atk, gtk2, glib, fontconfig, freetype
 , libgnomecanvas, libgnomeprint, libgnomeprintui
 , pango, libX11, xproto, zlib, poppler
-, autoconf, automake, libtool, pkgconfig}:
```

Comment:
Why was this changed? Seems like it's the same package list, but the end-bracket's on a different line.